axual-webclient/application_deployment_data.go

```diff
@@ -0,0 +1,159 @@
+package webclient
+
+type ApplicationDeploymentCreateResponse string
+
+type ApplicationDeploymentUpdateResponse interface{}
```

Comment:
ApplicationDeploymentUpdateResponse is empty interface because the update response is "204 No Content". In other words: the response is just HTTP signal 204 but no other content.